ZIGSIMPlus/Models/InAppPurchaseFacade.swift

```diff
@@ -59,16 +59,16 @@ class InAppPurchaseFacade: NSObject {
         guard let id = (Bundle.main.infoDictionary?["ProductIDForPremiumFeatures"] as? String) else {
             fatalError("Product ID not defined")
         }
-        
+
         return id
     }
 }
 
 extension InAppPurchaseFacade: SKProductsRequestDelegate {
-    func productsRequest(_ request: SKProductsRequest, didReceive response: SKProductsResponse) {
+    func productsRequest(_: SKProductsRequest, didReceive response: SKProductsResponse) {
```

Comment:
Will this delegate method called properly? 
I'm not sure.
If YES, it's OK.

Reply:
uh, I think it doesn't work... will fix them.
SwiftLint can't detect delegate methods .https://github.com/nicklockwood/SwiftFormat/issues/83

Reply:
I tested, and they seem to be called!